src/zope/app/apidoc/codemodule/browser/menu.pt

```diff
@@ -22,13 +23,13 @@
 
     <div tal:define="classes view/findClasses"
          tal:condition="classes">
-
-      <a href="" target="main"
-         tal:repeat="info classes"
-         tal:attributes="href info/url"
-         tal:content="info/path">
-        /zope/app/Application
-      </a>
+      <tal:omit-tag tal:repeat="info classes">
```

Comment:
Why was the 'tal:repeat' moved? Can it go back the way it was?

Reply:
Because link elements are inline, the class finder links don't each get their own line without additional help. This leads to some short paths sharing a single line (especially if the side frame is expanded) which can be confusing.
![image](https://user-images.githubusercontent.com/11380394/43649173-138ce4e8-9702-11e8-8af6-7ef2d96e48f1.png)
 `static_menu.pt` fixes this by wrapping the repeated link in a div containing the `tal:repeat` attribute. For `menu.pt` I didn't see the point in wrapping the links in a lot of useless divs like in its static counterpart (which uses the divs for the search function as well), so I used line break tags instead. Either way, the `tal:repeat` attribute needs to be moved to a block container around the link in order to make them show up on individual lines.
![image](https://user-images.githubusercontent.com/11380394/43649279-5cb029dc-9702-11e8-9eb9-6cc21591882d.png)

Reply:
D'oh, of course. I completely spaced on the `<br />` tag hiding down there. Maybe it should be on its own line so it's more visible.

Reply:
Alright, I moved it and updated `CHANGES.rst`.